AutoWorkup/utilities/configFileParser.py

```diff
@@ -34,6 +34,15 @@
 from .distributed import modify_qsub_args
 from . import misc
 
+# http://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python
+def str2bool(v):
+  if str(v).lower() in ("yes", "true", "t", "1"):
+      return True
+  elif str(v).lower() in ("no", "false", "f", "0"):
+      return False
+  print( "ERROR: INVALID String to bool conversion for '{0}'".format(v) )
+  raise
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The raise statement is not inside an except clause](https://www.codacy.com/app/hans-j-johnson/BRAINSTools/file/2704433953/issues/source?bid=3194574&fileBranchId=3342661#l44)